sdk/data/azcosmos/README.md

```diff
@@ -0,0 +1,157 @@
+# Microsoft Azure Cosmos DB SDK for Go, Golang
+
+## Introduction
+
+This client library enables client applications to connect to Azure Cosmos via the SQL API. Azure Cosmos is a globally distributed, multi-model database service. 
+
+## Getting Started
+
+### Prerequisites
+
+* Go versions 1.16 or higher
+* An Azure subscription or free Azure Cosmos DB trial account
+
+Note: If you don't have an Azure subscription, create a free account before you begin.
+You can Try Azure Cosmos DB for free without an Azure subscription, free of charge and commitments, or create an Azure Cosmos DB free tier account, with the first 400 RU/s and 5 GB of storage for free. You can also use the Azure Cosmos DB Emulator with a URI of https://localhost:8081. For the key to use with the emulator, see Authenticating requests.
+
+### Create an Azure Cosmos account
+
+You can create an Azure Cosmos account using:
+
+* [Azure Portal](https://portal.azure.com).
+* [Azure CLI](https://docs.microsoft.com/cli/azure).
+* [Azure ARM](https://docs.microsoft.com/azure/cosmos-db/quick-create-template).
+
+#### Install the package
+
+* Install the Azure Cosmos DB SDK for Go with `go get`:
+
+  ```bash
+  go get -u github.com/Azure/azure-sdk-for-go/sdk/data/azcosmos
+  ```
+  
+#### Authenticate the client
+
+In order to interact with the Azure CosmosDB service you'll need to create an instance of the Cosmos Client class. To make this possible you will need an URL and key of the Azure CosmosDB service.
+
+#### Create CosmosClient
+
+```go
+endpoint := "someEndpoint"
+key := "someKey"
+cred, _ := NewSharedKeyCredential(key)
+client, err := NewCosmosClient(endpoint, cred, nil)
+handle(err)
+```
+
+## Examples
+
+The following section provides several code snippets covering some of the most common CosmosDB SQL API tasks, including:
+* [Create Cosmos Client](#create-cosmos-client "Create Cosmos Client")
+* [Create Database](#create-database "Create Database")
+* [Create Container](#create-container "Create Container")
+* [CRUD operation on Items](#crud-operation-on-items "CRUD operation on Items")
+
+### Create Cosmos Client
+
+```go
+const (
+    CosmosDbEndpoint = "someEndpoint"
+    CosmoDbKey = "someKey"
+)
+
+cred, _ := NewSharedKeyCredential(CosmosDbEndpoint)
+client, err := NewCosmosClient(CosmoDbKey, cred, &CosmosClientOptions{})
+handle(err)
+```
+
+### Create Database
+
+Using the client created in previous example, you can create a database like this:
+
+```go
+database := CosmosDatabaseProperties{Id: dbName}
+response, err := client.CreateDatabase(context, database, nil, nil)
+handle(err)
+```
+
+### Create Container
+
+Using the above created database for creating a container, like this:
+
+```go
+response, err := database.CreateContainer(context, properties, throughput, nil)
```

Comment:
We are missing what is properties, what is throughput. We need to show how to create this variables